modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf

```diff
@@ -103,13 +97,24 @@
 			<c:choose>
 				<c:when test='<%= msvv.getFieldName() == "expirationDate" %>'>
 					<liferay-ui:message key="please-enter-a-valid-expiration-date-only-future-dates-are-accepted" />
+					<% isShowingExpirationError = true; %>
```

Comment:
We could assign this value at the scriplet above and use the same flag to test the `c:when` tag. Something like that:
```
<%
if (Validator.isNull(fieldLabelValue)) {
	fieldLabelValue = msvv.getFieldName();
}

isShowingExpirationError = msvv.getFieldName() == "expirationDate";
%>

<c:choose>
	<c:when test='<%= isShowingExpirationError %>'>
                 <liferay-ui:message key="please-enter-a-valid-expiration-date-only-future-dates-are-accepted" />
        </c:when>
	<c:otherwise>
```

Reply:
Good ideia, thanks!!